test/models/vote_test.rb

```diff
@@ -0,0 +1,9 @@
+require "test_helper"
+
+describe Vote do
```

Comment:
You really should have a test that ensures that the combination of User and Work is unique, so that the same user can't vote on the same work.